docs/fundamentals/executor/executor-in-flow.md

```diff
@@ -0,0 +1,462 @@
+# Executors inside a Flow
```

Comment:
This chapter is much more comprehensive than it used to be, i like that a lot. But maybe we need to denote which parts are absolutely essential and used all the time, and which parts are slightly more advanced / under the hood? I worry about overwhelming new users

Reply:
Suggestions are welcome, I am also not sure if it belongs here or in Flow section

Reply:
I have stared at documentation for so long now, my brain is a bit fried, can't come up with something rn. Will give it a shot on monday morning

Reply:
Appreciate it!